stdlib/testing/testdata/show_all_tag_keys.flux

```diff
@@ -1138,7 +1138,7 @@ t_show_all_tag_keys = (table=<-) =>
 		|> group()
 		|> distinct()
 		|> map(fn: (r) =>
-			(r._value)))
+			({_value: r._value})))

```

Comment:
We should convert this map call to a `keep` call since its purpose is to drop all columns but the `_value` column.

Reply:
Yeah, I guess you are right!  I made a no-look change 😅